guides/micronaut-microservices-distributed-tracing-zipkin-opentelemetry/metadata.json

```
@@ -0,0 +1,22 @@
+{
+  "title": "Use OpenTelemetry with Zipkin and the Micronaut Framework for Microservice Distributed Tracing",
+  "intro": "Use Zipkin distributed tracing to investigate the behaviour of your Micronaut applications.",
```

```suggestion
  "intro": "Use Zipkin distributed tracing to investigate the behavior of your Micronaut applications.",
```

change to American spelling optional